app/src/sharedTest/java/org/oppia/android/app/player/exploration/ExplorationActivityTest.kt

```diff
@@ -546,7 +546,7 @@ class ExplorationActivityTest {
           withEffectiveVisibility(Visibility.VISIBLE)
         )
       )
-      onView(allOf(withText("EN"), withEffectiveVisibility(Visibility.VISIBLE)))
+      onView(allOf(withId(R.id.ivAudioLanguage), withEffectiveVisibility(Visibility.VISIBLE)))
```

Comment:
please add a screenshot showing these tests are passing on Espresso and Robolectric.

Reply:
added the screenshots in pr description but Espresso tests are not passing on AudioFragmentTest